packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -440,6 +446,14 @@ describe('AddresslistVoting', function () {
         ).to.equal(id);
       });
 
+      it('reverts on voting None', async () => {
+        await advanceIntoVoteTime(startDate, endDate);
+
+        await expect(voting.vote(id, VoteOption.None, false))
```

Comment:
Just to be safe, I'd first try if it votes with Yes, to make sure proposal is votable and then try to vote it from another with None. The reason for this is VoteCastForbiden might have thrown out for different reason and this will trick us.